algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
Those changes are just reformatting.
It would be best to change your Android Studio settings so that it does *not* reformat files automatically.